changelogs/fragments/149_plugin_import_issue_fix.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- Fixed action plugins import non-existing module from ansible.netcommon issue (https://github.com/ansible-collections/community.network/issues/149)
```

Comment:
I don't think a changelog fragment is needed since the collection hasn't been released since the imports were added.